examples/eda-setup.yml

```diff
@@ -80,7 +80,7 @@
       ansible.eda.event_stream:
         organization_name: "{{ organization_name }}"
         name: basic event stream
-        credential_name: basic event stream
+        credential_name: basic event stream credential
         event_stream_type: Basic Event Stream
```

Comment:
Can you also remove this line? It is not necessary and can confuse users.

Reply:
done